service/src/main/kotlin/no/nav/su/se/bakover/service/behandling/BehandlingServiceImpl.kt

```diff
@@ -176,4 +177,8 @@ internal class BehandlingServiceImpl(
                 )
             }
     }
+
+    override fun slettBehandlingForBehandling(søknadId: UUID, avsluttetBegrunnelse: AvsluttetBegrunnelse) {
+        return behandlingRepo.slettBehandlingForBehandling(søknadId, avsluttetBegrunnelse)
```

Comment:
Alle ting som gjøres på en behandling burde vel en tur innom behandlingen for å se til at tilstanden er gyldig? Kanskje dette burde være en tilstand en behandling kan være i, det virker nesten sånn? Her kan vi vel kanskje se for oss et tilfelle hvor vi tilfeldigvis kaller denne funksjonen for en behandling som er under utbetaling, med den konsekvens at denne behandlingen aldri vil kunne hentes opp fra basen igjen?

Reply:
Er en god tankte det 👍